frontend/src/app/components/wp-table/timeline/header/wp-timeline-header.directive.ts

```diff
@@ -37,9 +37,10 @@ import {WorkPackageViewTimelineService} from "core-app/modules/work_packages/rou
 
 
 export const timelineHeaderCSSClass = 'wp-timeline--header-element';
+export const timelineHeaderSelector = 'wp-timeline-header';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/4e3d50b2e6e2cd56df848aec33dd3f111711a3d3/frontend/src/app/components/wp-table/timeline/header/wp-timeline-header.directive.ts#L41"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>